### PR TITLE
saving and loading added.

### PR DESCRIPTION
**Pull request by laptopadventure**

### CODE REVIEW

**Review by harfrancois — APPROVED** [review]

File: js/game.js [hunk 1]
```diff
@@ -1,79 +1,129 @@
...
+  return new Player(recieved);
 }
 
+function Player(savedata){
+  //should always be clear, who wants popups from last page?
+  this.popups = [];
+  if(!savedata) {
+    ///first time setup
+    if(window.location.pathname != '/index.html') {
+      console.log(window.location.pathname)
```
Comment (laptopadventure) [inline js/game.js]: remove this log

File: js/game.js [hunk 2]
```diff
@@ -1,79 +1,129 @@
 'use strict';
 
+//entry to the dom
 const main = document.querySelector('main');
+//main object- see Player doc
+const player = load();
 
+///saves game state.
+function save(){
+  player.hintCooldown = player.hintSystem.
```
Comment (laptopadventure) [inline js/game.js]: finish this (need to make a date object when the timer started)

**Review by laptopadventure — COMMENTED** [review]
my mini review for... myself

File: js/game.js [hunk 3]
```diff
@@ -1,79 +1,133 @@
...
+function Items(name, collected, page, x, y) {
   this.name = name;
+  this.collected = collected;
+  this.page = page;
   this.src = `images/${name}.png`;
   this.x = x;
   this.y = y;
   this.render = function() {
+    //unrender the old if it exists
+    let found = document.querySelector(`#${this.name}`);
```
Comment (Dcastro99) [inline js/game.js:79]: "#" on line 79, is that looking for an id?

**Review by Dcastro99 — CHANGES_REQUESTED** [review]
I think that's the only thing I can find. Just wasn't sure and couldn't find anything online about it. Overall I kinda get what you've done lol probably need it explained to me. Still learning.

**Review by denekm — APPROVED** [review]

**Review by Dcastro99 — APPROVED** [review]

**Pull request merged**